packages/node_modules/@webex/internal-plugin-devices/README.md

```diff
@@ -0,0 +1,42 @@
+# @webex/internal-plugin-devices
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Plugin for device management
+
+This is an internal Cisco Webex plugin. As such, it does not strictly adhere to semantic versioning. Use at your own risk. If you're not working on one of our first party clients, please look at our [developer api](https://developer.webex.com/getting-started.html) and stick to our public plugins.
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+## Install
+
+```bash
+npm install --save @webex/internal-plugin-devices
+```
+
+## Usage
+
+```js
+import '@webex/internal-plugin-devices';
+
+import WebexCore from '@webex/webex-core';
+
+const webex = new WebexCore();
+webex.internal.devices
```

Comment:
Feel free to add some sample functions here.

Reply:
Will do.